.github/ISSUE_TEMPLATE/improve-existing-docs.yaml

```
@@ -0,0 +1,49 @@
+name: Improve existing content
+description: Make a suggestion to improve the content in an existing article
+title: '[What I really want the title to be]'
+labels:
+  - content
+body:
+  - type: markdown
+    attributes:
+      value: |
+        **HUBBERS!! THIS IS GITHUB/DOCS OPEN SOURCE REPO. OPEN AN ISSUE IN GITHUB/DOCS-CONTENT INSTEAD.**
+
+        * For questions, ask in [Discussions](https://github.com/github/docs/discussions)
+        * Before you file an issue read the [Contributing guide](https://github.com/github/docs/blob/main/CONTRIBUTING.md)
+        * Check to make sure someone hasn't already opened a similar [issue](https://github.com/github/docs/issues)
+
+  - type: checkboxes
+    id: terms
+    attributes:
+      label: Code of Conduct
+      description: By submitting this issue, you agree to follow our [Code of Conduct](https://example.com)
+      options:
+        - label: I agree to follow this project's Code of Conduct
+          required: true
+
+  - type: textarea
+    attributes:
+      label: What article on docs.github.com is affected?
+      description: |
+        Please link to the article you'd like to see updated
```

```suggestion
      description: Please link to the article you'd like to see updated.
```

Since this is a single line, you don't really need the pipe.